src/lib/draw/AttributeMode.svelte

```diff
@@ -29,12 +30,7 @@
           "interventions-polygons",
         ],
       });
-      // TODO ? syntax
-      var newHoverId = null;
-      if (results.length > 0) {
-        newHoverId = results[0].id;
-      }
-      mapHover.set(newHoverId);
+      mapHover.set((results[0]?.id as number) || null);
```

Comment:
Kind of awkward, but this is as clean as I can make it. The `as number` is necessary because the ID might be `string` according to MapLibre's API. But because we only deal with numeric IDs, the cast is safe